prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -186,7 +191,7 @@ static int fi_ibv_rdm_av_insert(struct fid_av *av_fid, const void *addr,
 				fi_addr[i] = (uintptr_t) (void *) conn;
 				break;
 			case FI_AV_TABLE:
-				fi_addr[i] = av->count + i;
+				fi_addr[i] = av->used;
 				av->domain->rdm_cm->conn_table[fi_addr[i]] = conn;
```

Comment:
This seems to be the point at which connections are added to the internal table. However, it is dependent on fi_addr being non-NULL. In FI_AV_TABLE mode, since the insertions are done in a deterministic manner, it is okay for the user to pass NULL for this argument, which will cause the internal table to not be filled with the connections.